gdnative-core/src/lib.rs

```diff
@@ -53,13 +52,12 @@ pub mod thread_access;
 #[doc(hidden)]
 pub mod private;
 
-pub use crate::new_ref::NewRef;
-pub use crate::object::{AsArg, GodotObject, Instanciable, Null, QueueFree, Ref, TRef};
-
-pub use sys::GodotApi;
+//
+// Re-exports
+//
 
-#[doc(inline)]
-pub use error::GodotError;
+pub use new_ref::NewRef;
+pub use object::{GodotObject, Ref};
```

Comment:
Maybe we want to un-hide the `object` module so the other types that were originally exposed can be accessed without using `prelude::*`?

Also might be a good idea to re-export `TRef` here as well?

Reply:
Added some more things that are commonly used.